### DISCUSSION TITLE
Can I compare links between two environments?

### DISCUSSION THREAD

**Discussion by buger** [stevenvachon/broken-link-checker#217] [General]
The use case is pretty simple:

We just rolled out a new design with a big re-structure. 

One of our goals from SEO point of view that all *old* links from previous version will still work in new version (with http redirects and etc). 

So ideally I want to run a job to get list of links from my old website, and check this list against new website version.

Is something like this possible to do with this project? Or maybe by writing some extension.

Thank you!

**Comment by stevenvachon** [comment]
You could check both sites and filter the results by the resolved base URLs and maybe also redirected URLs.
